examples/gds-sessions.ipynb

```diff
@@ -0,0 +1,392 @@
+    "    (annie)-[:LIKES]->(banana),\n",
+    "    (matt)-[:LIKES]->(mango),\n",
+    "    (jeff)-[:LIKES]->(mango),\n",
+    "    (brie)-[:LIKES]->(banana),\n",
+    "    (elsa)-[:LIKES]->(plum),\n",
+    "    (john)-[:LIKES]->(plum),\n",
+    "  \n",
+    "    (dan)-[:KNOWS]->(annie),\n",
+    "    (dan)-[:KNOWS]->(matt),\n",
+    "    (annie)-[:KNOWS]->(matt),\n",
+    "    (annie)-[:KNOWS]->(jeff),\n",
+    "    (annie)-[:KNOWS]->(brie),\n",
+    "    (matt)-[:KNOWS]->(brie),\n",
+    "    (brie)-[:KNOWS]->(elsa),\n",
+    "    (brie)-[:KNOWS]->(jeff),\n",
+    "    (john)-[:KNOWS]->(jeff);\n",
+    "\"\"\"\n",
+    "\n",
+    "# making sure the database is actually empty\n",
+    "assert gds.run_cypher(\"MATCH (n) RETURN count(n)\").squeeze() == 0\n",
+    "\n",
+    "# let's now write our graph!\n",
+    "gds.run_cypher(data_query)\n",
+    "\n",
+    "gds.run_cypher(\"MATCH (n) RETURN count(n) AS nodeCount\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Projecting Graphs\n",
+    "\n",
+    "Now that we have imported a graph to our database, we can project it into our GDS Session.\n",
+    "We do that by using the `gds.graph.project()` endpoint."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from graphdatascience.session import GdsPropertyTypes\n",
+    "\n",
+    "G, result = gds.graph.project(\n",
+    "    \"people-and-fruits\",\n",
+    "    \"\"\"\n",
+    "    CALL {\n",
+    "        MATCH (p1:Person)\n",
+    "        OPTIONAL MATCH (p1)-[r:KNOWS]->(p2:Person)\n",
+    "        RETURN p1 AS source, r AS rel, p2 AS target, p1 {.age, .experience, .hipster } AS sourceNodeProperties, p2 {.age, .experience, .hipster } AS targetNodeProperties\n",
+    "        UNION\n",
+    "        MATCH (f:Fruit)\n",
+    "        OPTIONAL MATCH (f)<-[r:LIKES]-(p:Person)\n",
+    "        RETURN p AS source, r AS rel, f AS target, p {.age, .experience, .hipster } AS sourceNodeProperties, f { .tropical, .sourness, .sweetness } AS targetNodeProperties\n",
+    "    }\n",
+    "    RETURN gds.graph.project.remote(source, target, {\n",
+    "      sourceNodeProperties: sourceNodeProperties,\n",
+    "      targetNodeProperties: targetNodeProperties,\n",
+    "      sourceNodeLabels: labels(source),\n",
+    "      targetNodeLabels: labels(target),\n",
+    "      relationshipType: type(rel)\n",
+    "    })\n",
+    "    \"\"\",\n",
+    "    nodePropertySchema={\n",
+    "        \"age\": GdsPropertyTypes.LONG,\n",
+    "        \"experience\": GdsPropertyTypes.LONG,\n",
+    "        \"hipster\": GdsPropertyTypes.LONG,\n",
+    "        \"tropical\": GdsPropertyTypes.LONG,\n",
+    "        \"sourness\": GdsPropertyTypes.DOUBLE,\n",
+    "        \"sweetness\": GdsPropertyTypes.DOUBLE,\n",
+    "    },\n",
+    "    relationshipPropertySchema={},\n",
+    ")\n",
+    "\n",
+    "str(G)"
```

Comment:
if we just write `G` here. The executed cell should show its str version anyway

Reply:
No, the formatted string of `G` is very verbose and shows the whole configuration including the query. Wrapping this in `str(G)` produces a nicer output.